AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_entity_utils.py

```diff
@@ -313,6 +313,42 @@ def get_type_ids(component_names: list, entity_type: EditorEntityType = EditorEn
             bus.Broadcast, "FindComponentTypeIdsByEntityType", component_names, entity_type.value)
         return type_ids
 
+    def is_property_visible(self, component_property_path: str, expected: bool = True) -> bool:
```

Comment:
Should this method go on the base `EditorComponent`? This seems generally useful

Reply:
@cgalvan @smurly should we get a backlog item for this?